[Draft] medium less

TomNUSDS opened the pull request:
- Add [README](https://github.com/usds/website/tree/tomn-uses/medium-less/_blog_posts) to blog posts directory and exclude from jekyll transpile. **<-- Start reading here**
- Create a new blog post related template.
- Modify News and Blog page to show blog posts.
- Port over several posts from Medium. There are 238 posts total in medium.
- Fix css compile issue
- Update Jekyll's exclude list in the config.

**TODO:**
  - Images in blog posts are still linked back to Medium.
  - Some posts have doule images as I'm trying to figure out which approach to use for embedding them (markdown or `<picture>` or `<img>`).

TomNUSDS commented on assets/stylesheets/uswds/styles.css:
I don't know why the .css is checked into git.

TomNUSDS commented on assets/stylesheets/application.css:
I removed this file from git trying to make it regenerate, but everything seems to work without it? Is it a dangling, intermediate file?

TomNUSDS commented on _blog_posts/README.md:
This file is excluded from Jekyll's collection by adding it to the _config.yml. We could potentially add more instructions for the different sections this way.

drew-usds commented:
@TomNUSDS I think I fixed our setup _and_ our SCSS watch problems with [a98d8a3](https://github.com/usds/website/pull/895/commits/a98d8a324972a739a1f2229a2b47cc934d52554b), [3561642](https://github.com/usds/website/pull/895/commits/3561642427a41b75987a166214d487e7c06a7e03), and [af9d82e](https://github.com/usds/website/pull/895/commits/af9d82e97579580acd04f811aa236fd1e1e89a30).

- Updated Ruby version to 3.1.1
- Added webrick to gemfile

☝️ Fixes issues where we were forcing local installs of gems (ex `jekyll`) despite being included in the dependencies for `github-pages`. Also adds `webrick` -- isn't default part of ruby >=3 anymore -- for running with NPM instead of Docker.

- Adds `css:sass:watch` to package.json and edits `serve` to use that instead of `css:sass`. Build processes/etc are updated with new paths.
- Renames and moves scss and scss partials around a bit so the watch script can work correctly.

☝️ Fixes problem where scss wasn't compiling on changes during local development. New watch script is looking at all scss in `/assets/stylesheets/*` for changes and outputting them to `/assets/stylesheets/styles.css`.

TomNUSDS committed:
Update README.md

TomNUSDS commented on README.md:
```diff
@@ -8,24 +8,28 @@ The USDS website is built with:
 Before getting started, install the following on your system:
 
-- [Ruby](https://www.ruby-lang.org/en/documentation/installation/), best set up on version 2.7 for Jekyll
+- [Ruby](https://www.ruby-lang.org/en/documentation/installation/)(v 3.1.1)
```
```suggestion
- [Ruby](https://www.ruby-lang.org/en/documentation/installation/)(v 3.2.2)
```

TomNUSDS committed:
Update README.md

update ruby version in readme

TomNUSDS commented on .github/workflows/deploy.yml:
```diff
@@ -44,52 +44,52 @@ jobs:
         uses: ruby/setup-ruby@ee2113536afb7f793eed4ce60e8d3b26db912da4 # v1.127.0
         with:
-          ruby-version: '2.7' # Not needed with a .ruby-version file
+          ruby-version: "3.1.1" # Not needed with a .ruby-version file
```
```suggestion
          ruby-version: "3.2.2" # Not needed with a .ruby-version file
```

TomNUSDS commented on documentation/2023-refresh/engineering-resources.md:
```diff
@@ -1,27 +1,31 @@
    ruby-3.0.5 [ x86_64 ]
 =* ruby-3.1.3 [ x86_64 ]
-$ rvm  install ruby-2.7
+$ rvm  install ruby-3.1.1
```
```suggestion
$ rvm  install ruby-3.2.2
```

TomNUSDS committed:
Update broken image

Extra slash we missed removing

amwhitty commented on _news-and-blog/2022-08-15-a-quick-guide-to-inclusive-design.md:
```diff
@@ -0,0 +1,221 @@
+
+### 1. Design with users, not for them
+
+One of the [six USDS values](/mission#our-values) is to “design with users, not for them.” What’s implicit in that is that designing _with_ users means _all_ users regardless of their ability, context, or demographics. We’re designing technology and services for a diverse population so our research participants should be equally diverse. So, not only do we strive to do research with diverse popul ...
```
mission link fails on staging b/c it's missing `/website-staging` to file path

amwhitty commented on _news-and-blog/2022-08-15-a-quick-guide-to-inclusive-design.md:
```diff
@@ -0,0 +1,221 @@
+
+### 5. Hire and empower great people
+
+Another one of the [six USDS values](/mission#our-values) is to “hire and empower great people.”
```
same as above, missing `website-staging` in URL

amwhitty commented on _news-and-blog/2022-08-15-a-quick-guide-to-inclusive-design.md:
```diff
@@ -0,0 +1,221 @@
+  e.g., “[progressive disclosure](https://www.uxmatters.com/mt/archives/2020/05/designing-for-progressive-disclosure.php)”
+  to let complex information unfold and “wizards” to present one question at a time that result in a custom message for
+  their situation — a great example of this is
+  the [VA discharge status tool](/projects/discharge-upgrade-tool)).
```
same as above missing `/website-staging` in filepath

amwhitty commented on _news-and-blog/2022-08-15-a-quick-guide-to-inclusive-design.md:
```diff
@@ -0,0 +1,221 @@
+  show them their longer content on a small screen, the low usability of this becomes unavoidable.
+- Remember that internal users benefit from mobile responsive tools too because they may shrink multiple windows down to fit on their screen side-by-side.
+
+![Squares made of dark blue, medium blue, gold and white triangles are piled on top of each other to form a pyramid of squares. This represents the USDS value of "hire and empower great people."](2022-08-15-a-quick-guide-to-inclusive-design-img/rVj8FA.png)
```
image is not loading

TomNUSDS replied:
Fixed in next update. It had a leading `/` and shouldn't have. (Note: staging is not in sync with main)

amwhitty commented on _news-and-blog/2022-08-15-a-quick-guide-to-inclusive-design.md:
```diff
@@ -0,0 +1,221 @@
+
+**It’s always nice to end with a quote, so here’s one from a user research participant to one of our designers:**
+
+_“As a disabled Veteran it means \_a lot_ to me that this type of site is being built up for us! It already appears to be
```
Pull quote is not formatting if it's supposed to match pull quote style on project pages.

drew-usds replied:
This was just in the body of the text in the OG post, not a pull quote. It looks like it has some extra trailing characters on it which I'll fix!

amwhitty reviewed (commented):
5 flags for the quick guide to inclusive design post.

amwhitty commented on _news-and-blog/2023-10-17-solutions-by-families-for-families.md:
```diff
@@ -0,0 +1,250 @@
+permalink: /news-and-blog/solutions-by-families-for-families-4b8150
+---
+
+_This blog is part two in a four-part series that highlights our work on [Having a Child and Early Childhood Life Experience](https://www.performance.gov/cx/life-experiences/having-a-child-and-early-childhood-for-low-income-families/). You can read the first post, “Why We’re Designing Government to Work Better and Smarter for Families” on [our blog](/blog/why-were-designing-government-to-work-bet ...
```
update second link for `our blog` to: https://usds.github.io/website-staging/news-and-blog/why-were-designing-government-to-work-better-and-smarter-for-families-dee5c2

amwhitty commented on _news-and-blog/2023-10-17-solutions-by-families-for-families.md:
```diff
@@ -0,0 +1,250 @@
+work. Brought together by the Customer Experience Executive Order, teams are tackling the extraordinary challenge of
+modernizing and implementing government services that are simple, accessible, easy to use, and equitable for families._
+
+_Read the team’s first post, “Why We’re Designing Government to Work Better and Smarter for Families” on [our blog](/blog/why-were-designing-government-to-work-better-and-smarter-for-families)._
```
Same comment from line 24:
update link for our blog to: https://usds.github.io/website-staging/news-and-blog/why-were-designing-government-to-work-better-and-smarter-for-families-dee5c2

amwhitty commented on _news-and-blog/2023-10-17-solutions-by-families-for-families.md:
```diff
@@ -0,0 +1,250 @@
+
+<blockquote class="pullquote" markdown="1">
+Solutions are never implemented and change never comes. There’s inquiries and roundtables, but is it really just for information-sake or is it because we’re really going to do something in the next term?
+<footer>– Community Health Worker in South Carolina who supports new moms</footer>
```
quote attribution doesn't show up in reading mode. Not sure if this matters.

drew-usds replied:
Interesting! I'm getting inconsistent results trying to replicate/resolve this, but looks like it might have to do with the tags nested within `<blockquote>`?

Made a bug fix issue for it here: https://github.com/usds/website-management/issues/152

amwhitty reviewed (commented):
2 broken links and one question about quote attribution in reading mode.

amwhitty commented on _news-and-blog/2023-03-29-reducing-burden-of-child-care-assistance-applications.md:
```diff
@@ -0,0 +1,107 @@
+
+_If you’d like to work on projects like this, consider joining USDS! We’re hiring mission-driven engineers, product
+managers, designers, bureaucracy hackers, procurement specialists, and operations experts who want to make an impact on
+the lives of their fellow Americans. [Apply here](/apply)._
```
broken link try: https://usds.github.io/website-staging/apply

amwhitty commented on _news-and-blog/2023-03-29-reducing-burden-of-child-care-assistance-applications.md:
```diff
@@ -0,0 +1,107 @@
+
+![Graphic image of large text that reads "USDS Value: Design with users, not for them."](2023-03-29-reducing-burden-of-child-care-assistance-applications-img/g6RgdQ.png)
+
+In [keeping with our value](/mission#our-values) to “design with users, not for them,” we made sure to not only speak
```
link not working. https://usds.github.io/website-staging/mission#our-values

amwhitty commented on _news-and-blog/2023-04-07-why-we-serve-bill-chapman.md:
```diff
@@ -0,0 +1,92 @@
+**_If you’d like to work on these important projects, join us!_**
+
+_**We’re hiring mission-driven talent professionals, engineers, designers, product managers, bureaucracy hackers,
+procurement specialists, and operations pros who want to make an impact on the lives of their fellow Americans. [Apply here](/apply)**_.
```
apply link broken: https://usds.github.io/website-staging/apply

amwhitty commented on _news-and-blog/2023-08-11-usds-marks-nine-years-of-transforming-government-services.md:
```diff
@@ -0,0 +1,190 @@
+
+Nine years ago, the United States Digital Service, or “USDS,” got our start as a small team based at the White House to
+change how Government delivers services in a digital age. Since August 11, 2014, we’ve been laser-focused
+on [our mission](/mission) to deliver better Government services to the American people through
```
broken link, suggest https://usds.github.io/website-staging/mission

amwhitty commented on _news-and-blog/2023-08-11-usds-marks-nine-years-of-transforming-government-services.md:
```diff
@@ -0,0 +1,190 @@
+of important tasks such as checking benefits, booking appointments, messaging healthcare providers, and looking at lists of VA payments.
+
+Since its launch, over one-million Veterans have used the app, and it’s garnered nearly perfect ratings in the app
+store, with over 80,000 five-star reviews. _Find out more about this project [on our blog](/blog/meeting-veterans-where-they-are-with-accessible-mobile-tech)._
```
broken link, suggest
https://usds.github.io/website-staging/news-and-blog/meeting-veterans-where-they-are-with-accessible-mobile-tech-fba73d

amwhitty commented on _news-and-blog/2023-08-11-usds-marks-nine-years-of-transforming-government-services.md:
```diff
@@ -0,0 +1,190 @@
+Assistance Application, which will help reduce burdens and break down the barriers for families that need child care
+assistance.
+
+_Read more about this work [on our blog](/blog/reducing-burden-of-child-care-assistance-applications) and in [our projects](https://www.usds.gov/projects/acf)._
```
broken blog link. suggest: https://usds.github.io/website-staging/news-and-blog/reducing-burden-of-child-care-assistance-applications-e595d4

TomNUSDS replied:
```suggestion
_Read more about this work [on our blog](/blog/reducing-burden-of-child-care-assistance-applications-e595d4) and in [our projects](https://www.usds.gov/projects/acf)._
```

amwhitty commented on _news-and-blog/2023-08-11-usds-marks-nine-years-of-transforming-government-services.md:
```diff
@@ -0,0 +1,190 @@
+year in Medicare claims, implementing the Quality Payment Program, launching the No Surprises Act webpage; and promoting the importance of seamless and straightforward digital services at the agency.
+
+_Read more about one of CMS’s
+projects in [our projects](/projects/medicare-payment-program)._
```
broken link, suggest: https://usds.github.io/website-staging/projects/medicare-payment-program

amwhitty commented on _news-and-blog/2023-08-11-usds-marks-nine-years-of-transforming-government-services.md:
```diff
@@ -0,0 +1,190 @@
+This supports ongoing work to ensure that Government can appropriately support the communities and people most
+negatively impacted by the changing climate.
+
+_Read more about this program and our work [on our blog](/blog/tackling-the-climate-crisis-with-open-source)._
```
broken link, suggest: https://usds.github.io/website-staging/news-and-blog/tackling-the-climate-crisis-with-open-source-00a52a

TomNUSDS replied:
```suggestion
_Read more about this program and our work [on our blog](/blog/tackling-the-climate-crisis-with-open-source-00a52a)._
```

amwhitty commented on _news-and-blog/2023-08-11-usds-marks-nine-years-of-transforming-government-services.md:
```diff
@@ -0,0 +1,190 @@
+- The **Federal Notification Service** helps families stay connected to Government services at crucial moments (e.g.,
+  when it’s time to enroll or renew benefits) through text messages.
+
+_Learn more about this work [on our blog](/blog/why-were-designing-government-to-work-better-and-smarter-for-families)
```
broken blog link, suggest: https://usds.github.io/website-staging/news-and-blog/why-were-designing-government-to-work-better-and-smarter-for-families-dee5c2

TomNUSDS replied:
```suggestion
_Learn more about this work [on our blog](/blog/why-were-designing-government-to-work-better-and-smarter-for-families-dee5c2)
```

TomNUSDS replied:
```suggestion
_Learn more about this work [on our blog](/blog/why-were-designing-government-to-work-better-and-smarter-for-families-dee5c2)
```

amwhitty commented on _news-and-blog/2023-04-07-why-we-serve-bill-chapman.md:
```diff
@@ -0,0 +1,92 @@
+# partial path to image
+carousel_image: /news-and-blog/2023-04-07-why-we-serve-bill-chapman-img/carousel-why-we-serve-chapman.jpg
+# accessibility text for image
+carousel_image_alt_text: ""
```
add alt text: USDSer Bill Chapman wears a backpack standing at a cliff overlook with crashing waves.

drew-usds replied:
Fixed!

amwhitty commented on _news-and-blog/2023-03-29-reducing-burden-of-child-care-assistance-applications.md:
```diff
@@ -0,0 +1,107 @@
+# partial path to image
+carousel_image: /news-and-blog/2023-03-29-reducing-burden-of-child-care-assistance-applications-img/carousel-reducing-burden-card.jpg
+# accessibility text for image
+carousel_image_alt_text: ""
```
add alt text: Photograph of two women and two young children playing at a table covered in wooden blocks. Text overlaid on the image reads &quot;Reducing child care assistance burdens for families.&quot;"

drew-usds replied:
Fixed!

TomNUSDS committed:
Update .github/workflows/deploy.yml

Set ruby-version to 3.2.2 to be consistent with other places. WATCH FOR ISSUES.

TomNUSDS committed:
Update documentation/2023-refresh/engineering-resources.md

ruby v3.2.2 in documentation

TomNUSDS commented on _news-and-blog/README.md:
```diff
@@ -0,0 +1,78 @@
+dateline_str: "Nov 10, 2022"
+readtime_str: "5"
+byline_str: "U.S. Digital Service"
+permalink: /news-and-blog/meeting-veterans-where-they-are-with-accessible-mobile-tech
```
```suggestion
permalink: /news-and-blog/meeting-veterans-where-they-are-with-accessible-mobile-tech-fba73d
```

drew-usds committed:
fix _ italic formatting in blog post, AW flagged

drew-usds committed:
fix missing carousel img alt text

TomNUSDS committed:
fix blog paths

TomNUSDS committed:
Double "by"

drew-usds reviewed (approved):
LGTM! 👍
